framework/utils/xmlUtils.py

```diff
@@ -206,16 +206,16 @@ def fixXmlText(msg):
     @ Out, msg, string, fixed string
   """
   #if not a string, pass it back through
-  if not isinstance(msg,basestring):
+  if not type(msg).__name__ in ['unicode','str']:
```

Comment:
I did try .utils, and that did not solve the problem when running from make_docs.sh  I don't think it would be hard to solve, I just only tried the first thing I could think of.